src/util/java/mcMMO/PotionConfigGenerator.java

```diff
@@ -104,6 +104,8 @@ private static String getMCName(PotionType type) {
                 case WEAKNESS :
                 case TURTLE_MASTER:
                 case SLOW_FALLING:
+                case INFESTED:
```

Comment:
This won't be compatible with older versions, you'll have to change the switch case to operate on string value